<launch>
<node name="uav_msg_unpack_center" pkg="physical_message_unpack" type="physical_message_unpack" output="screen">
      <param name="iot_msg_sub_topic" value="/msg_from_iot"/>
      <param name="local_position_msg_pub_topic" value="/phsical_uav/local/position"/>
      <param name="velocity_msg_pub_topic" value="/physical_uav/velocity"/>
      <param name="dt_object_position_recv_hz" value="30.0"/>
      <param name="v2p_delay_time_msg_pub_topic" value="/v2p_delay/time"/>
      <param name="p2v_delay_time_msg_pub_topic" value="/p2v_delay/time"/>
      <param name="update_model_msg_pub_topic" value="/updata_model/info"/>
      <param name="object_move_position_pub_topic" value="/virtual/uav0/current_position"/>
  </node>
</launch>
